hooks/post-commit.py

```diff
@@ -5,7 +5,7 @@
 
 
 def run_command(command):
-    return subprocess.check_output(command.split(), shell=True).rstrip()
+    return subprocess.check_output(command.split()).rstrip()
```

Comment:
no worries -- I have no idea why I did this originally (I first wrote this code in 2010)... so lets take it out and see what happens. This isn't rocket science.